### DISCUSSION TITLE
Error: Time limit reached

### DISCUSSION THREAD

**Discussion by alanhe421** [coder/code-server#5788] [Q&A]
I create ssh-http-agent to proxy code-server-web. Time limit reached is Often occur.

So I want to know how long the Time limit is? any prop to setting it?

![image](https://user-images.githubusercontent.com/9245110/202479286-2275d3cc-92ce-41da-813e-5a7752df6fd8.png)

## version info
- code-server@v4.8.3

**Comment by jsjoeio** [comment]
Hmm 🤔 @code-asher might know. I'm not totally sure to be honest. If you follow the StackTrace to `remoteAgentConnection`, does it show anything helpful?

**Reply by code-asher** [reply]
I think there are a few timeouts.  The reconnect one seems to be 30s while the handshake is 10s.  None of them appear to be configurable.
